apps/i18n/signup/en_us.json

```diff
@@ -63,5 +63,9 @@
   "display_name_error_message": "Please enter your display name",
   "email_error_message": "Please enter a valid email",
   "age_error_message": "Please select an age",
-  "state_error_message": "Please select a state"
+  "state_error_message": "Please select a state",
+  "data_transfer_notice": "Data transfer notice:",
```

Comment:
Nit: can we add an asterisk here like the other required questions please? I'm worried it won't be clear to the user which question is blocking the submit button from being enabled without marking it somewhere

Reply:
Ideally I don't think we would modify the strings themselves but I did add the requirement tag via styling. Good callout- I've added this to both views 
<img width="567" alt="Screenshot 2024-09-30 at 11 06 01 AM" src="https://github.com/user-attachments/assets/2f5d4c80-e6d7-497c-838f-f7abe98387ec">
<img width="527" alt="Screenshot 2024-09-30 at 11 10 42 AM" src="https://github.com/user-attachments/assets/03eab8b8-f6e6-44fd-9e3e-98c3c46f1d67">

Reply:
Perfect thank you!